src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/PollsnackWidgetPageObject.java

```diff
@@ -11,12 +13,17 @@
 
   @FindBy(css = "iframe[data-wikia-widget=\"pollsnack\"]")
   private WebElement pollsnackIframe;
+  @FindBy(css = "iframe[data-wikia-widget=\"pollsnack\"]")
+  private List<WebElement> pollsnackIframeList;
   @FindBy(css = "iframe")
   private WebElement pollsnackBody;
 
   private static final String TAG_NAME = "pollsnack";
   private static final String ARTICLE_NAME = "PollsnackWidget";
-  private static final String TAG = "<pollsnack hash=\"q7kiw9kz\"/>";
+  private static final String TAGS[] = {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move the array designator from the variable to the type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1197)